tests/store/test_store_api.py

```diff
@@ -60,11 +60,11 @@ def test_delete_notondisk(store, tmpdir, bundle_data):
         bundle_data_notondisk['name']='angrybird'
 
         bundle_obj, version_obj = store.add_bundle(data=bundle_data)
-        store.add_commit(bundle_obj)
         include_version(tmpdir, version_obj, hardlink=False)
+        store.add_commit(bundle_obj)
```

Comment:
I suggest we add GIVEN/WHEN/THEN in this test to improve readability and maintainability, or what do you think?

Reply:
I've added these comments now in this whole test file